cpp/inline/mcts/Manager.inl

```diff
@@ -30,9 +33,10 @@ inline Manager<Game>::Manager(const ManagerParams& params, NNEvaluationServiceBa
     init_profiling_dir(profiling_dir.string());
   }
 
-  // TODO: if no_model, construct a UniformNNEvaluationService here
   if (service) {
     nn_eval_service_ = service;
+  } else if (params.no_model) {
+    nn_eval_service_ =  new UniformNNEvaluationService<Game>();
   } else if (!params.no_model) {
     nn_eval_service_ = NNEvaluationService::create(params);
   } else if (!params.model_filename.empty()) {
```

Comment:
This look buggy to me. As written, the final `else if` clause will never get reached. Please re-think the logical truth-table here.